components/Layout.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+// Looks for width prop to be either 'fixed', or 'fluid'
+const Layout = props => {
+  const { width, children } = props;
+  return <StyledDiv width={width}>{children}</StyledDiv>;
+};
+
+export default Layout;
+
+Layout.propTypes = {
+  width: PropTypes.string,
```

Comment:
If we're only supporting two types of widths, could we refactor this to be a boolean? If we need to support more than two types of widths, what do you think about using [`PropTypes.oneOf`](https://reactjs.org/docs/typechecking-with-proptypes.html) to make sure we get a valid value?